scripts/notarize.js

```diff
@@ -0,0 +1,23 @@
+require('dotenv').config();
+const { notarize } = require('electron-notarize');
+
+exports.default = async function notarizing(context) {
+    const { electronPlatformName, appOutDir } = context;  
+    if (electronPlatformName !== 'darwin') {
+        return;
+    }
+
+    const appName = context.packager.appInfo.productFilename;
+    const appleId = process.env.APPLEID;
+    
+    if (!appleId) {
+        return;
+    }
+
+    return await notarize({
+        appBundleId: 'io.rancherdesktop.app',
```

Comment:
I assume this should match the value in `electron-builder.yml`?  (Possibly resolved by changing the other one, if this is better)